src/core/klsh.js

```
@@ -0,0 +1,106 @@
+        break;
+      }
+      default:
+        break;
+    }
+  }
+  return { text, stderr };
+}
+
+// Shell entrypoint: execute parsed commands from stdin
+function main({ args = [], stdin = '', env: parentEnv = {} }) {
+  let stdout = '';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'stdout' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)